### DISCUSSION TITLE
Some clarification on how it works

### DISCUSSION THREAD

**Discussion by Erd89** [AdguardTeam/AdGuardHome#6518] [General]
By passing all DNS requests to the local server, this only takes care of not resolving (replying with 0.0.0.0) for the domains listed as to be blocked but all those to be resolved are resolved by the Adguard public server or not? Thanks in advance 😛

**Comment by ainar-g** [comment]
The upstream servers are configurable, see [the documentation](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams).

**Reply by Erd89** [reply]
Thanks a lot for the answer. Very clear :)
